frontend/src/routes/Governance/common/util.tsx

```diff
@@ -537,3 +550,19 @@ export function handlePolicyAutomationSubmit(
     history.push(window.history?.state?.state?.from ?? NavigationPath.policies)
   })
 }
+
+export function formatDescriptionForDropdown(desc: string) {
+  const formattedDescription = []
+  if (desc) {
+    const descLines = desc.split(/\r?\n/)
+    for (let i = 0; i < descLines.length; i++) {
+      const line = descLines[i]
+      formattedDescription.push(<span>{line}</span>)
+      if (i < descLines.length - 1) {
+        formattedDescription.push(<br></br>)
```

Comment:
```suggestion
        formattedDescription.push(<br>)
```

Reply:
If we do the paragraph tag then you probably don't need these break HTML elements.